### ISSUE TITLE
Missing parts of post's text when using WP Gitbuber MD

### ISSUE THREAD

**Issue by trejder** [terrylinooo/githuber-md#51]
For some reason, for some blog posts WP Gitbuber MD is not displaying actual post's text or it is stripping parts of the text.

An example, how it should look and how it looks when WP Gitbuber MD is disabled:

![gitbubber-3](https://user-images.githubusercontent.com/2903392/56653396-e599e200-668d-11e9-92ed-bfe6af64ffd6.jpg)

And this is, how it looks when it is enabled:

![gitbubber-4](https://user-images.githubusercontent.com/2903392/56653415-ecc0f000-668d-11e9-96f4-1ed6b17109ad.jpg)

The [original post](http://itfuckup.com/2019/04/19/jira-cloud-mobile-has-arrived/) for reference. Using WP Gitbuber 1.9.1.

**Please, consider this as high-priority. With missing text we cannot use your extension because we are not sure when it is going to delete parts of our blog posts from database, since it is not displaying the whole text when editing**.

**Comment by terrylinooo** [comment]
This feature is for specific posts you don't want to use Markdown, allows you switch your Markdown editor to Rich editor (or Gutenberg)
Be careful of using this feature - If you switch back to Rich editor then save, Markdown text will be cleared.

**Comment by trejder** [comment]
I think you misunderstood my initial report (or I don't understand your answer). This issue has nothing to do with switching between your editor and Gutenberg.

This issue is about bug in your editor that causes that for certain posts to display only half of actual post content in your editor while after switching to Gutenberg -- whole text is displayed.
